kubernetes-model-generator/kubernetes-model-core/src/test/java/io/fabric8/kubernetes/api/model/QuantityTest.java

```diff
@@ -198,4 +198,40 @@ public void testIndexOfUnit() {
     assertEquals(4, Quantity.indexOfUnit("123 K"));
     assertEquals(4, Quantity.indexOfUnit("123c"));
   }
+
+  @Test
+  @DisplayName("Test fromAmountInBytes method")
+  public void testFromAmountInBytes() {
```

Comment:
Could you please make this test parameterized instead?

Reply:
thanks for the suggestion. I made the change.